tests.sh: check for required test/dev deps

If someone runs tests before installing dev dependencies they may encounter this:
```
./tests.sh: line 11: .: filename argument required
.: usage: . filename [arguments]
./tests.sh: line 17: env_parallel: command not found
ZIGPATH set to /home/pwndbg/pwndbg/.zig
[+] Building tls.i386.c
/home/pwndbg/pwndbg/.zig/zig cc \
-Wall -DDEBUG=1 -ggdb -O0 -gdwarf-4 \
-target i386-linux-gnu \
-o tls.i386.out tls.i386.c
make: /home/pwndbg/pwndbg/.zig/zig: Command not found
make: *** [makefile:105: tls.i386.out] Error 127
```

The tests.sh could just check for some required binaries and tell the user to run setup-dev.sh beforehand

--- a/tests.sh
+++ b/tests.sh
@@ -1,5 +1,15 @@
 #!/bin/bash
 
+
+# Check some basic test dependencies
+if ! command -v env_parallel &> /dev/null
+then
+    echo 'Error: The `env_parallel` command could not be found. You should run `setup-dev.sh` to install development dependencies.'
+    echo '(Alternatively, run ./tests.sh with `--serial` to skip using parallel test running. However, if `env_parallel` is missing, it is likely that other dependencies like the `zig` compiler are also missing)'
+    exit
+fi
+
+
 # Run integration tests
 (cd tests/gdb-tests && ./tests.sh $@)
 

--- a/tests.sh
+++ b/tests.sh
@@ -1,14 +1,11 @@
 #!/bin/bash
 
-
 # Check some basic test dependencies
-if ! command -v env_parallel &> /dev/null
-then
+if ! command -v env_parallel &> /dev/null; then
     echo 'Error: The `env_parallel` command could not be found. You should run `setup-dev.sh` to install development dependencies.'
     echo '(Alternatively, run ./tests.sh with `--serial` to skip using parallel test running. However, if `env_parallel` is missing, it is likely that other dependencies like the `zig` compiler are also missing)'
     exit
 fi
-
 
 # Run integration tests
 (cd tests/gdb-tests && ./tests.sh $@)

